### ISSUE TITLE
www.seqan.de unreachable

### ISSUE THREAD

**Issue by james-d-mitchell** [seqan/seqan#2458]
The webpage is currently unavailable:

![Screenshot 2021-02-19 at 17 20 14](https://user-images.githubusercontent.com/6686071/108538479-be86b880-72d6-11eb-9495-c80ef8eca7ee.png)

**Comment by marehr** [comment]
TLDR; Website is down, because of security reasons and will hopefully be back early next week.

Do you use our website in any scripts and rely on its availability?

----
Long Explanation:

Yes, we know. It is down since 2021-02-15.

We are currently in the transition to host a "new" website based on jekyll (a static website generator), since we had a lot of troubles with our old website which was based on WordPress. WordPress had a lot of security alerts in recent time, and we generally aren't experts with WordPress, so we decided to make a switch to a different back-end. The new website is not complete yet, but I think we will be able to copy over the content in a fairly short time.

You can access 
* a mirrored version of our old website at https://marehr.github.io/old.seqan.de/ and 
* a preview of our new website at https://www-seqan-de.vercel.app

in the meantime.

We made the old website only accessible via our intranet (= Freie Universität) to circumvent any exploits. We would have made the new website available already, but our DNS-Server is centrally administered by our IT department, so we (=seqan-team) don't have any access to it. We are in contact with them, and I think they will change the DNS to our new website early next week.

**Comment by james-d-mitchell** [comment]
Thanks for the reply @marehr, I don't have anything that relies on the availability of the site, I was just clicking around and noticed that it was down, and wasn't sure if you knew or not.

**Comment by marehr** [comment]
@james-d-mitchell Thank you for reporting ❤️

**Comment by marehr** [comment]
Issue moved to [seqan/www.seqan.de #12](https://github.com/seqan/www.seqan.de/issues/12) via [**ZenHub**](https://www.zenhub.com/)

**Comment by marehr** [comment]
Issue moved to [seqan/www.seqan.de #13](https://github.com/seqan/www.seqan.de/issues/13) via [**ZenHub**](https://www.zenhub.com/)
